### PR TITLE
Allowing to test multiple callbacks on the same task

### PR DESCRIPTION
**Pull request by ghost**
In my project I had several 'after' callbacks with the same task. This is part of my recipe:

after 'development', 'load_dinamic_variables'
after 'qa', 'load_dinamic_variables'
after 'prod', 'load_dinamic_variables'

when I tried to test this callbacks I found out that all test that asserted the 'after' callbacks but the first one, failed. I realized that this was a bug in capistrano-spec actually since when you seek the callbacks you use the first one in the array and discard the rest.

I modified this in order to check if the callback is present in the whole array instead of just checking the first one.

Added some tests as well.

### CODE REVIEW

File: lib/capistrano/spec.rb [hunk 1]
```diff
@@ -1,3 +1,5 @@
+require 'pry'
```
Comment (technicalpickles) [inline lib/capistrano/spec.rb]: This line is causing the build to fail :sob:

Comment (petems) [inline lib/capistrano/spec.rb]: Shouldn't be too big an issue, just add it to the Gem file and it should be fine :+1:

Comment (technicalpickles) [inline lib/capistrano/spec.rb]: pry is a debugging thing though, and not actually used anywhere in this diff

Comment (petems) [inline lib/capistrano/spec.rb]: Ah, OK prob best remove it then :smile_cat:

**Comment by ghost** [comment]
Hi guys, any news regarding this PR. Do you think it is ok or something else needs to be corrected?

**Comment by petems** [comment]
Hi @jbscv, sorry I didn't see you've already removed the `pry` require, this looks good to go and fixes the issue when I tested locally. Funnily enough I thought I had seen this before, but I thought I'd just written my config wrong, so good catch! :smile: 

If no-one has any other issues with this, I'll merge it tomorrow morning :+1:

File: lib/capistrano/spec.rb [hunk 2]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (petems) [inline lib/capistrano/spec.rb]: Not a dealbreaker, but could you remove this whitespace?

**Comment by ghost** [comment]
done

**Pull request merged**